app/experimenter/experiments/tests/api/v5/test_queries.py

```diff
@@ -1251,6 +1251,37 @@ def test_branches(self):
                 experiment_data["treatmentBranches"],
             )
 
+    @parameterized.expand([[True], [False]])
+    def test_prevent_pref_conflicts(self, prevent_pref_conflicts):
```

Comment:
I like to use test names that include like, the action that it's doing, so like `test_query_prevent_pref_conflicts`, but since this is just looking up the value of the field, you could just fold that into another test that's already querying the rest of the experiment?